### PR TITLE
Add new make command to publish the application to a running network

### LINKED ISSUE #201: Add new make command to publish the data reconciliation application to a running CCF network

### PR DESCRIPTION
**Pull request by Aymalla**
closes #201

### CODE REVIEW

File: scripts/submit_proposal.sh [hunk 1]
```diff
@@ -48,21 +48,31 @@ done
...
-
+# proposal submitter vote for proposal
 /opt/ccf_virtual/bin/scurl.sh "$network_url/gov/proposals/$proposal0_id/ballots" --cacert $service_cert --signing-key $signing_key --signing-cert $signing_cert --data-binary @${app_dir}/governance/vote/vote_accept.json -H "content-type: application/json" | jq
 
+for ((i = 1 ; i < $member_count ; i++)); do
+  signing_cert="$certificate_dir/member${i}_cert.pem"
+  signing_key="$certificate_dir/member${i}_privk.pem"
+  /opt/ccf_virtual/bin/scurl.sh "$network_url/gov/proposals/$proposal0_id/ballots" --cacert $service_cert --signing-key $signing_key --signing-cert $signing_cert --data-binary @${app_dir}/governance/vote/vote_accept.json -H "content-type: application/json" | jq
+done
+
```
Comment (ross-p-smith) [inline scripts/submit_proposal.sh:75]: Can we just have 1 new line at the end?

**Review by ross-p-smith — APPROVED** [review]

File: scripts/submit_proposal.sh (hunk 1 above)
Comment (Aymalla) [reply scripts/submit_proposal.sh]: Done

File: data-reconciliation-app/Makefile [hunk 2]
```diff
@@ -88,4 +88,14 @@ deploy-mccf: ## 🚀 Deploy Managed CCF
 	@cd .. && $(MAKE) deploy-mccf
 
 deploy-ms-idp: ## 🔐 Create an Identity Provider
-	@cd .. && $(MAKE) deploy-ms-idp
\ No newline at end of file
+	@cd .. && $(MAKE) deploy-ms-idp
+
+publish-app-sandbox: build ## 🧪 Deploy the Data Reconciliation Application to a network
```
Comment (macromania) [inline data-reconciliation-app/Makefile:93]: I wonder if this command is better when we call it `publish-app-to-sandbox`?

File: data-reconciliation-app/Makefile [hunk 3]
```diff
@@ -88,4 +88,14 @@ deploy-mccf: ## 🚀 Deploy Managed CCF
...
+	@echo -e "\e[34m$@\e[0m" || true
+	@. ../scripts/submit_proposal.sh --network-url "https://127.0.0.1:8000" --proposal-file ./dist/set_js_app.json --certificate_dir "./workspace/sandbox_common" --member-count 2
+	@. ../scripts/submit_proposal.sh --network-url "https://127.0.0.1:8000" --proposal-file ./workspace/proposals/set_jwt_issuer_ms_proposal.json --certificate_dir "./workspace/sandbox_common" --member-count 2
+
+publish-app-mccf: build ## 🧪 Deploy the Data Reconciliation Application to a network
```
Comment (macromania) [inline data-reconciliation-app/Makefile:98]: I wonder if this command is better when we call it `deploy-app-to-mccf`?

File: data-reconciliation-app/Makefile (hunk 2 above)
Comment (Aymalla) [reply data-reconciliation-app/Makefile]: Done

File: data-reconciliation-app/Makefile (hunk 3 above)
Comment (Aymalla) [reply data-reconciliation-app/Makefile]: Done

**Pull request merged**